fbw-a32nx/src/systems/fmgc/src/efis/EfisSymbols.ts

```diff
@@ -78,6 +79,8 @@ export class EfisSymbols<T extends number> {
 
   private lastEfisInterfaceVersions: Record<EfisSide, number> = { L: -1, R: -1 };
 
+  private minRwyLengthDisplay: number = parseInt(NXDataStore.get('CONFIG_MIN_RWY_LENGTH', '1500'));
```

Comment:
Here the default value is 1500, but in [fbw-common/src/systems/instruments/src/EFB/Settings/Pages/AircraftOptionsPinProgramsPage.tsx](https://github.com/flybywiresim/aircraft/pull/9458/files#diff-d316475b8413cbbdc58316f48ae0b80eb70cd558c4f46152264a46c8ccab5b81) the default value is 3000.